src/main/java/com/iota/iri/zmq/ZmqTransactionStatsPublisher.java

```diff
@@ -0,0 +1,134 @@
+package com.iota.iri.zmq;
+
+import com.iota.iri.controllers.TipsViewModel;
+import com.iota.iri.controllers.TransactionViewModel;
+import com.iota.iri.model.Hash;
+import com.iota.iri.service.tipselection.TipSelector;
+import com.iota.iri.storage.Tangle;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.time.Duration;
+import java.time.Instant;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Optional;
+import java.util.concurrent.atomic.AtomicBoolean;
+
+/**
+ * Publishes the number of total transactions and confirmed transactions to the ZeroMQ.
+ *
+ * Only transactions are counted that have an arrival time between 5 minutes and 2 hours in the past.
+ * 
+ * For the confirmed transactions, the normal tip selection is performed to determine a supertip. The number of
+ * transactions in its past set is then published to ZMQ.
+ */
+public class ZmqTransactionStatsPublisher {
```

Comment:
What do you think about renaming to `transactionStatsPublisher` ? I feel the zmq prefix makes it sound like it handles all the zmq output or that it's strongly attached to zmq, whereas this class's real purpose as I understand it is to gather stats and publish (on whatever outlet)

Reply:
I agree! `ZmqTransactionStatsPublisher` is not great. Currently, the new classes are in the `zmq` package. Should they be moved, even if we change the name?

Reply:
I think this should be a service, probably best to create a stats package? (I know IRI are also looking at moving stats from API)

Reply:
`TransactionStatsPublisher` has been renamed and moved to `iri.service.stats`